docs/Commands.md

```diff
@@ -378,6 +378,12 @@ Searching for books with "Python" in any TEXT attribute, returning the price sto
 FT.SEARCH books-idx "python" RETURN 3 $.book.price AS price
 ```
 
+Searching for books with textual similarity to some book using VECTOR attribute, returning the top 10 results, sorted by distance.
```

Comment:
this one is not holding yet, without `SORTBY text_vec_score`

Reply:
Right. But I included `SORTBY text_vec_score`. Are you suggesting to remove this example?